pom.xml

```diff
@@ -29,7 +29,7 @@
     
     <artifactId>org.apache.sling.caconfig.spi</artifactId>
     <packaging>jar</packaging>
-    <version>1.3.5-SNAPSHOT</version>
+    <version>1.4.0-SNAPSHOT</version>
```

Comment:
please keep it to the original version, we will update the version only during the release.